src/plugins/intel_gpu/src/runtime/format.cpp

```diff
@@ -0,0 +1,434 @@
+            { g_os_is_zyx_osv16_isv16,                     { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "goizyx", "oixyz?g",  {{0, 16}, {1, 16}}}},
+            { g_os_zyx_is_osv16_isv4,                      { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "gozyxi", "oixyz?g",  {{0, 16}, {1, 4}}}},
+            { g_os_zyx_is_osv16_isv16,                     { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "gozyxi", "oixyz?g",  {{0, 16}, {1, 16}}}},
+            { g_os_zyx_is_osv16_isv32,                     { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "gozyxi", "oixyz?g",  {{0, 16}, {1, 32}}}},
+            { g_os_zyx_is_osv32_isv4,                      { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "gozyxi", "oixyz?g",  {{0, 32}, {1, 4}}}},
+            { g_os_zyx_is_osv32_isv16,                     { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "gozyxi", "oixyz?g",  {{0, 32}, {1, 16}}}},
+            { g_os_zyx_is_osv32_isv32,                     { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "gozyxi", "oixyz?g",  {{0, 32}, {1, 32}}}},
+            { g_os_is_yx_osa4_isa8_osv8_isv4,              { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{0, 32}, {1, 32}}}},
+            { g_os_is_zyx_osa4_isa8_osv8_isv4,             { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "goizyx", "oixyz?g",  {{0, 32}, {1, 32}}}},
+            { g_os_is_yx_osa4_isa8_osv8_isv2,              { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{0, 32}, {1, 16}}}},
+            { g_os_is_zyx_osa4_isa8_osv8_isv2,             { 1, 1, 3, 1, {0, 1, 2, 3, 4, 5}, "goizyx", "oixyz?g",  {{0, 32}, {1, 16}}}},
+            { g_os_is_yx_osa2_isa8_osv16_isv4,             { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{0, 32}, {1, 32}}}},
+            { g_os_is_yx_osa2_isa8_osv16_isv2,             { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{0, 32}, {1, 16}}}},
+            { gs_oi_yxs_gsv4_yxsv4,                        { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{6, 4}}}},
+            { gs_oi_yxs_gsv16_yxsv4,                       { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{6, 16}}}},
+            { gs_oi_yxs_gsv32_yxsv4,                       { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{6, 32}}}},
+            { g_os_is_yx_isv16_osv16,                      { 1, 1, 2, 1, {0, 1, 2, 3, 4},    "goiyx",  "oixy??g",  {{1, 16}, {0, 16}}}},
+            { gi_yxs_os_yxsv2_osv16,                       { 1, 1, 2, 1, {0, 2, 3, 4, 1},    "giyxo",  "oixy??g",  {{0, 16}}}},
+            { giy_xs_os_xsv2_osv8__ao32,                   { 1, 1, 2, 1, {0, 2, 3, 4, 1},    "giyxo",  "oixy??g",  {{2, 2}, {0, 8}}}},
+            { giy_xs_os_xsv2_osv16__ao32,                  { 1, 1, 2, 1, {0, 2, 3, 4, 1},    "giyxo",  "oixy??g",  {{2, 2}, {0, 16}}}},
+    };
+    if (traits.find(fmt) == traits.end()) {
+        throw std::runtime_error("[clDNN] Format description is missing in fmt traits");
+    }
+    return traits.at(fmt);
+}
+
+std::string format::to_string() const {
+    switch (value) {
```

Comment:
Minor suggestion: how about set this "string" in format_trait too?

Reply:
Applied